File: android/src/main/kotlin/com/deeplinkly/flutter_deeplinkly/network/DomainConfig.kt
package com.deeplinkly.flutter_deeplinkly.network

/**
 * Domain configuration for API endpoints
 */
object DomainConfig {
    private const val BASE = "https://deeplinkly.com/api/v1"
    
    const val RESOLVE_CLICK_ENDPOINT = "$BASE/resolve"
    const val GENERATE_LINK_ENDPOINT = "$BASE/generate-url"
    const val ENRICH_ENDPOINT = "$BASE/enrich"
    const val ERROR_ENDPOINT = "$BASE/error/report"
}

File: ios/Classes/DomainConfig.swift
// DomainConfig.swift
enum DomainConfig {
    static let base = "https://warm-leopard-uniquely.ngrok-free.app"
    static let enrich = "\(base)/api/v1/enrich"
    static let sdkError = "\(base)/api/v1/sdk-error"
    static let resolveClick = "\(base)/api/v1/resolve"
    static let generateLink = "\(base)/api/v1/generate-url"
}
